pkg/controllers/machine/garbagecollection/suite_test.go

```diff
@@ -96,7 +96,7 @@ var _ = Describe("MachineGarbageCollection", func() {
 		nodeTemplate := test.AWSNodeTemplate(v1alpha1.AWSNodeTemplateSpec{})
 		provisioner := test.Provisioner(coretest.ProvisionerOptions{
 			ProviderRef: &v1alpha5.MachineTemplateRef{
-				APIVersion: v1alpha5.TestingGroup + "v1alpha1",
+				APIVersion: "testing.karpenter.sh/v1alpha1",
```

Comment:
Do we need to change this?
```suggestion
				APIVersion: "testing/v1alpha1",
```